wrapper.py

```diff
@@ -0,0 +1,132 @@
+        for element in elements:
+            if element.text == locator_and_text[1]:
+                element.click()
+
+    def clear_element(self, locator, text_value='default'):
+        """Clears the element with the specific text_value"""
+        WebDriverWait(self.driver, self.default_timeout).until(EC.presence_of_element_located(locator))
+        self.get_element(locator).clear()
+
+    def send_keys(self, locator, keys, text_value='default'):
+        """Send keys to the element with the the specific text_value"""
+        WebDriverWait(self.driver, self.default_timeout).until(EC.presence_of_element_located(locator))
+        elements = self.get_elements(locator)
+        if text_value == 'default':
+            elements[0].send_keys(keys)
+        else:
+            for element in elements:
+                if element.text == text_value:
+                    element.send_keys(keys)
+
+    def get_attr_value(self, locator, attr):
+        """Get attribute value of the element"""
+        WebDriverWait(self.driver, self.default_timeout).until(EC.element_to_be_clickable(locator))
+        element = self.get_element(locator)
+        return element.get_attribute(attr)
+
+    def company_view_update_delete(self, locator1, locator2, company_name):
+        """This function clicks on company details/update/delete buttons
+         according to the company name and specific locators"""
+        tbody = self.driver.find_elements(*locator1)
+        for i in tbody:
+            if company_name in i.text:
+                td = i.find_element(*locator2)
+                td.click()
+
+    def read_data_in_textbox(self, locator_list, locator_attribute):
+        """Gets values from the input fields by attribute and return a list of this values"""
+        data_list = []
+        for el in range(len(locator_list)):
+            a = self.driver.find_element_by_id(locator_list[el]).get_attribute(
+                locator_attribute)
+            data_list.append(a)
+        return data_list
+
+    def get_text_of_element(self, locator):
+        return self.get_element(locator).text
+
+    def pause(self, time):
+        webdriver.support.wait.time.sleep(time)
+
+    def click_element_double_locator(self, locator_wait, locator_element):
+        """This function takes two locators, first one for 'WebDriverWait', the second one for click on the element"""
+        WebDriverWait(self.driver, self.default_timeout).until(EC.visibility_of_element_located(locator_wait))
+        element = self.get_element(locator_element)
+        element.click()
+
+    def click_button_change(self, locator_buttons, locator_wait):
+        """Special function for press 'Change' button"""
+        WebDriverWait(self.driver, self.default_timeout).until(EC.visibility_of_element_located(locator_wait))
+        buttons = self.get_elements(locator_buttons)
+        change = None
+        for i in buttons:
+            if i.text == 'Change':
+                change = i
```

Comment:
why not
```suggestion
                i.click()
```
?
and, please, rename `i`

Reply:
will you come today?

Reply:
sorry, I hurried and did not see this comment, but I will come tomorrow or you may write questions to me in slack)